apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -1685,6 +1762,267 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
+                        }
+                        placeholder={
+                            t("console:develop.features.applications.forms.inboundOIDC.sections" +
+                                ".clientAuthentication.fields.authenticationMethod.placeholder")
+                        }
+                        listen={ (values: Map<string, FormValue>) => handleAuthMethodChange(values) }
+                        children={ getAllowedList(metadata.tokenEndpointAuthMethod) }
+                        readOnly={ readOnly }
+                    />
+                    <Hint>
+                        { t("console:develop.features.applications.forms.inboundOIDC.sections" +
+                            ".clientAuthentication.fields.authenticationMethod.hint") }
+                    </Hint>
+                </Grid.Column>
+            </Grid.Row>
+
+            { selectedAuthMethod == "private_key_jwt" &&
+                (
+                    <Grid.Row columns={ 1 }>
+                        <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
+                            <Field
+                                ref={ tokenEndpointAuthSigningAlg }
+                                name="tokenEndpointAuthSigningAlg"
+                                label={
+                                    t("console:develop.features.applications.forms.inboundOIDC.sections" +
+                                        ".clientAuthentication.fields.signingAlgorithm.label")
+                                }
+                                required={ false }
+                                type="dropdown"
+                                disabled={ false }
+                                default={
+                                    initialValues?.clientAuthentication ?
+                                        initialValues.clientAuthentication.tokenEndpointAuthSigningAlg
+                                        : metadata.tokenEndpointSignatureAlgorithm.defaultValue
+                                }
+                                placeholder={
+                                    t("console:develop.features.applications.forms.inboundOIDC.sections" +
+                                        ".clientAuthentication.fields.signingAlgorithm.placeholder")
+                                }
+                                children={ getAllowedList(metadata.tokenEndpointSignatureAlgorithm) }
+                                readOnly={ readOnly }
+                            />
+                            <Hint>
+                                { t("console:develop.features.applications.forms.inboundOIDC.sections" +
+                                    ".clientAuthentication.fields.signingAlgorithm.hint") }
+                            </Hint>
+
+                        </Grid.Column>
+                    </Grid.Row>
+                )
+            }
+
+            { selectedAuthMethod == "tls_client_auth" &&
```

Comment:
Consider moving "tls_client_auth" to a constant

Reply:
fixed with [4838970](https://github.com/wso2/identity-apps/pull/4284/commits/4838970b7dcb77e09e551d9946f3a84075499656)